src/layout/menu.jsx

```diff
@@ -3,30 +3,34 @@ import { Link } from "gatsby";
 import styled from "styled-components";
```

Comment:
Now, menu is not necessarily a layout (beecause it doesn't wrap the content of the page) so I would probably keep this in the components directory